go/src/vendors/sonic/client/dbclient/redis_provider.go

```diff
@@ -0,0 +1,87 @@
+package dbclient
+
+import (
+        . "go/src/lib/lomcommon"
```

Comment:
suggestion to remove . and use lomcommon.<>

Reply:
Done.